draft-ietf-quic-transport.md

```diff
@@ -2352,17 +2337,56 @@ Application Error Code:
   {{app-error-codes}}).
 
 
+## PATH_CHALLENGE Frame {#frame-path-challenge}
```

Comment:
Should these fall after ACK frames in terms of ordering?

Reply:
I don't believe we have specified packet ordering if that's what you're referring to, so I'd prefer not to start now.

Reply:
Ah no, sorry, this is *much* more minor: I'm referring to ordering within the document itself (although that might not be a concern at this point), which no longer matches the table above.

Reply:
We've already done one reorder of the document to match the table -- we'll probably need to do another as part of the grand editorial pass being discussed in a few months.

Reply:
Done